src/vm-vcpu/src/vcpu/mod.rs

```diff
@@ -288,7 +291,12 @@ impl KvmVcpu {
                                 eprintln!("Failed to read from serial port");
                             }
                         } else {
-                            // Read from some other port.
+                            // Emulate a dummy i8042.
+                            if data.len() == 1 && addr == 0x64 {
+                                data[0] = 0x0;
+                            } else if data.len() == 1 && addr == 0x61 {
+                                data[0] = 0x20;
```

Comment:
Can you please add a comment explaining why `0x20`?

Reply:
@andreeaflorescu can you please help with a bit of context?

Reply:
This is what kvmtool does as well, didn't really do a dive deep.